inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/resource/sink/cls/ClsResourceOperator.java

```diff
@@ -113,6 +120,32 @@ private void createTopicID(SinkInfo sinkInfo) {
         }
     }
 
+    private String getTopicID(SinkInfo sinkInfo, ClsDataNodeDTO clsDataNode, ClsSinkDTO clsSinkDTO)
+            throws TencentCloudSDKException {
+        String topicID = describeTopicIDByTopicName(sinkInfo, clsDataNode);
+        if (Strings.isEmpty(topicID)) {
+            // if topic don't exist,create topic in cls
```

Comment:
if topic don't exist, create topic in cls

Reply:
done